specs/merge/beacon-chain.md

```diff
@@ -263,21 +263,9 @@ def initialize_beacon_state_from_eth1(eth1_block_hash: Bytes32,
     # Set genesis validators root for domain separation and chain versioning
     state.genesis_validators_root = hash_tree_root(state.validators)
 
-    # [New in Merge] Construct execution payload header
-    # Note: initialized with zero block height
-    state.latest_execution_payload_header = ExecutionPayloadHeader(
-        block_hash=eth1_block_hash,
-        parent_hash=Hash32(),
-        coinbase=Bytes20(),
-        state_root=Bytes32(),
-        number=uint64(0),
-        gas_limit=uint64(0),
-        gas_used=uint64(0),
-        timestamp=eth1_timestamp,
-        receipt_root=Bytes32(),
-        logs_bloom=ByteVector[BYTES_PER_LOGS_BLOOM](),
-        transactions_root=Root(),
-    )
+    # Initialize the execution payload header (with block number set to 0)
```

Comment:
Maybe do it once to avoid duplication if we'd need to initialize more fields in the future?
```suggestion
    # [New in Merge] Initialize the execution payload header (with block number set to 0)
```

Reply:
done :)